### DISCUSSION TITLE
Error when executing main python script

### DISCUSSION THREAD

**Discussion by kevinkandziora** [steinerthomas/easy2homeassistant#9] [General]
Hey,

first of all, thank you for your work. Unfortunatelly I got the following error:

`2024-12-01 17:32:12,126 - xml_validator - ERROR - /var/folders/mh/kv8246px737f7_1t1vss1l_r0000gn/T/tmpulu7livk/configuration/Channels.xml:2:0:ERROR:SCHEMASV:SCHEMAV_ELEMENT_CONTENT: Element 'config': This element is not expected. Expected is ( property ).
2024-12-01 17:32:12,141 - __main__ - ERROR - Scheme validation of '/var/folders/mh/kv8246px737f7_1t1vss1l_r0000gn/T/tmpulu7livk/configuration/Channels.xml' failed!`

I am not sure how to solve this issue. Can you help me with this issue? If you need more information, do not hesitate to ask for it.

Thank you in advance!
Cheers, Kevin

**Comment by steinerthomas** [comment]
Hi,

I can help you. It seems that your configuration is different do my genereated xml scheme. I've added  scheme validation because I'm not sure if other versions/projects lead to different xml configurations.

Can you please provide me your txa project file (would be nice for future analysis to have a second project available), or at least the Channels.xml and the Products.xml file in a private GitHub repo or send it via mail?

Thanks!
Thomas

**Reply by steinerthomas** [reply]
Problem caused by missing (optional) property "Last-Modified" e.g. `<property key="Last-Modified" value="1729265022123" type="long"/>`
Fixed with: https://github.com/steinerthomas/easy2homeassistant/pull/10
